recipes/colombo/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "v4.0" %}
+
+package:
+  name: colombo
+  version: {{ version }}
+
+source:
+  url: https://github.com/brinkmanlab/colombo/releases/download/{{ version }}/Colombo.zip
+  sha256: 0f3e2ecb62897b7ea38e77f8b5c5cfab562f3af8f49ce5cf30172b8844b278fe
+  
+requirements:
+  run:
+    - openjdk
+
+test:
+  commands:
+    - SigiHMM --help
+    - mSigiHMM --help
+    - SigiCRF --help
```

Comment:
You are not testing Colombo here. Would be nice to test this as well.

Reply:
Colombo provides no CLI and requires a X11 display and user input to test.